frontend/src/style/ThemeProvider.tsx

```diff
@@ -141,6 +145,12 @@ const defaultPalette = {
         },
         name: 'trainingPlanTaskComplete',
     }),
+    dullGrey: defaultTheme.palette.augmentColor({
+        color: {
+           main: '#E8E8E8', 
```

Comment:
I actually meant why not just use the color `white` instead of defining a new `dullGrey`, but I guess this is fine too

Reply:
yea it was meant a light grey color, white looked a bit too hard on eyes